taskcluster/ci/release-notify-testrail/kind.yml

```diff
@@ -0,0 +1,72 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+---
+loader: taskgraph.loader.transform:loader
+
+transforms:
+    - taskgraph.transforms.notify:transforms
+    - taskgraph.transforms.job:transforms
+    - taskgraph.transforms.task:transforms
+
+task-defaults:
+    description: Sends Slack message to release testers that Testrail Milestone was created.
+    worker-type: b-android
+    worker:
+        docker-image: {in-tree: ui-tests}
+        max-run-time: 120
+        env:
+            TESTRAIL_PROJECT_ID: '59' # Fenix Browser
+            TESTRAIL_TEST_SUITE_ID: '49319' # Test Automation Release Milestone
+    run:
+        use-caches: false
+        using: run-commands
+        secrets:
+            - name: project/mobile/ci/testrail
+              key: testrailCredentials
+              path: .testrail_credentials.json
+              json: true
+    routes:
+        - notify.slack-channel.G016BC5FUHJ.on-failed
+    scopes:
+        - queue:route:notify.slack-channel.G016BC5FUHJ # notify mobile-alerts-sandbox on failure
+        - notify:slack-channel:G016BC5FUHJ
+        - queue:route:notify.slack-channel.C02KDDS9QM9 # notify mobile-testeng on success
+        - notify:slack-channel:C02KDDS9QM9
+
```

Comment:
wanted to check with you @ahal , do these `notify` keys need to be indented under `queue`? not really sure if i need all of this since i'm not using `.on-failed` or `.on-succeed` here.

I'm only trying to add the necessary scopes so the python script can send a message to either channel depending on if the script passes or fails, not this task.

Reply:
No this is right, they should be at the same level.